index.html

```diff
@@ -11,12 +11,18 @@
     <h1>The Black Binder</h1>
     <h2>Your number one directory for Black Culture</h2>
     <p>Enter your email address and we will let you know when we are almost ready to go live</p>
-    <input type="text" name="Email" value="Your email address...">
-    <button type="button">Notify Me</button>
```

Comment:
Can you put this back in?

`<input type="text" name="Email" value="Your email address...">`
`<button type="button">Notify Me</button>`